packages/mui-material/src/PaginationItem/PaginationItem.js

```diff
@@ -41,6 +41,7 @@ const useUtilityClasses = (ownerState) => {
       `size${capitalize(size)}`,
       variant,
       shape,
+      ['primary', 'secondary'].includes(color) && `color${capitalize(color)}`,
```

Comment:
I think we should do

```suggestion
      color !== 'standard' && `color${capitalize(color)}`,
```

As the logic is "add a color* class for all color values except the default"

Reply:
got it!! fixed in this commit https://github.com/mui/material-ui/pull/40673/commits/84f7045702f7787f03d7d451f6ceb5185a143ecb